src/planwise/client/projects/components/sidebar.cljs

```diff
@@ -1,15 +1,35 @@
 (ns planwise.client.projects.components.sidebar
+  (:require-macros [reagent.ratom :refer [reaction]])
   (:require [re-frame.core :refer [subscribe dispatch]]
             [re-com.core :as rc]
             [planwise.client.components.progress-bar :as progress-bar]
             [planwise.client.components.filters :as filters]
-            [planwise.client.projects.db :as db]))
+            [planwise.client.projects.db :as db]
+            [planwise.client.utils :as utils]))
 
 
+(defn- demographic-stat [title value]
+  [:div.stat
+   [:div.stat-title title]
+   [:div.stat-value value]])
+
 (defn- demographics-filters []
-  [:div.sidebar-filters
-   [:div.filter-info
-    [:p "Filter here the population you are analyzing."]]])
+  (let [current-project (subscribe [:projects/current-data])
+        population (reaction (:region-population @current-project))
+        area (reaction (:region-area-km2 @current-project))
+        density (reaction (/ @population @area))]
+    (fn []
+      [:div.sidebar-filters
+       [:div.filter-info
+        ;; [:p "Filter here the population you are analyzing."]
+        [:div.demographic-stats
+         (demographic-stat "Area" [:span (utils/format (int @area)) " km" [:sup 2]])
+         (demographic-stat "Density" [:span (utils/format @density) " /km" [:sup 2]])
+         (demographic-stat "Total population" (utils/format @population))
+         ]
+        [:span.small
+         "Population information source: "
+         [:a {:href "http://www.worldpop.org.uk/"} "WorldPop"]]]])))
```

Comment:
Let's use this space to provide proper attribution according to [CCv4 license](https://creativecommons.org/licenses/by/4.0/). We'll probably need to cite also the author (Geodata institute of the university of Southampton), see if there is a suggested way to cite them in the worldpop site.

Also, let's change _population information source_ to _demographic data source_ to be consistent with the wording of the section.